### ISSUE TITLE
Space Host Policy doesn't load when service broker is using `admin` user identity

### ISSUE THREAD

**Issue by micahlee** [cyberark/conjur-service-broker#119]

**Comment by jtuttle** [comment]
I believe this is due to the fact that we [assume in the space host policy](https://github.com/cyberark/conjur-service-broker/blob/master/app/models/space_host_policy.rb#L48) that the identity that the service broker uses to authenticate with Conjur is a host.

**Comment by micahlee** [comment]
Yes, that seems to be correct per: https://github.com/cyberark/conjur-service-broker/blob/4d56f76f948ca2cc2d76fca642ff600bb1e38ca0/app/models/space_host_policy.rb#L48.

If we do decide to make this work with `user` as well as `host`, we do have the existing helper method here we can use: https://github.com/cyberark/conjur-service-broker/blob/2e3e7cbed7a6e25cca6e5c5fd442176068154261/lib/conjur_client.rb#L46

**Comment by izgeri** [comment]
@micahlee this is in the epic - is this still correct? or should this be left in the backlog?

**Comment by micahlee** [comment]
It's fine with me if it stays in the backlog for now. It's only an issue if the service broker is configured with a `user` role, and we don't suggest or demonstrate that anywhere.
